### PR TITLE
Fix booking data loss on refresh using sessionStorage

### PR DESCRIPTION
**Pull request by welvin21**

### CODE REVIEW

File: src/components/main/BookingSummary.js [hunk 1]
```diff
@@ -52,10 +52,16 @@ export const BookingSummary = () => {
...
+  const room = roomData.roomNumber
+    ? roomData.roomNumber
+    : sessionStorage.getItem('roomNumber');
+  const date = bookingData.bookingDate
+    ? bookingData.bookingDate
+    : sessionStorage.getItem('bookingDate');
+  const start = bookingData.start
+    ? bookingData.start
+    : sessionStorage.getItem('start');
+  const end = bookingData.end ? bookingData.end : sessionStorage.getItem('end');
```
Comment (utkarsh867) [inline src/components/main/BookingSummary.js:65]: This is not the right way to go. We definitely will use `apollo-cache-persist`

File: src/components/main/ChooseTime.js [hunk 2]
```diff
@@ -47,8 +47,12 @@ export const ChooseTime = () => {
...
   const { data: roomData, client } = useQuery(GET_ROOM_NUMBER);
   const { data: bookingData } = useQuery(GET_BOOKING_DATE);
-  const room = roomData.roomNumber;
-  const date = bookingData.bookingDate;
+  const room = roomData.roomNumber
+    ? roomData.roomNumber
+    : sessionStorage.getItem('roomNumber');
+  const date = bookingData.bookingDate
+    ? bookingData.bookingDate
+    : sessionStorage.getItem('bookingDate');
```
Comment (utkarsh867) [inline src/components/main/ChooseTime.js:55]: Same as before

File: src/components/main/ChooseTime.js [hunk 3]
```diff
@@ -66,6 +70,8 @@ export const ChooseTime = () => {
 
   const handleNextButtonClick = () => {
     if (validateTime(bookings, date, start, end)) {
+      sessionStorage.setItem('start', moment(start).toISOString());
+      sessionStorage.setItem('end', moment(end).toISOString());
```
Comment (utkarsh867) [inline src/components/main/ChooseTime.js:74]: Same as before

**Review by utkarsh867 — CHANGES_REQUESTED** [review]

**Comment by utkarsh867** [comment]
@welvin21 There is a flaw in this approach of yours to directly use `sessionStorage`. Instead, we will use `apollo-cache-persist` to save the data. We will fix the summary page instead because there is probably a bug on that page.

Gonna close this pull request because it rolls back all the changes that were made to make this work.

File: src/components/main/BookingSummary.js (hunk 1 above)
Comment (welvin21) [reply src/components/main/BookingSummary.js:65]: Alright, I'll see it again thenn

**Pull request closed without merging**